docker.d/configs/prod/worker.json

```diff
@@ -0,0 +1,13 @@
+{
+  "work_dir": "/app/workdir",
+  "artifact_dir": "/app/artifacts",
+  "hg": "hg",
+  "hg_share_base_dir": "/tmp/share_base",
+  "upstream_repo": "https://hg.mozilla.org/mozilla-unified",
+  "hg_ssh_user": {
+    "$eval": "SSH_USER"
+  },
+  "hg_ssh_keyfile": {
+    "$eval": "'/app/configs/ssh_key_${SSH_USER}'"
```

Comment:
I'm not sure if json-e will eval this properly, worth verifying before merging.

Reply:
i've checked this and it works

Reply:
Hmm, this is what I get in the generated file:
```
 "hg_ssh_keyfile": "/app/configs/ssh_key_${SSH_USER}",
```